packages/api/src/controllers/user.ts

```diff
@@ -326,23 +355,55 @@ app.post("/", validatePost("user"), async (req, res) => {
 
 app.patch(
   "/:id/suspended",
+  validatePost("suspend-user-payload"),
   authMiddleware({ anyAdmin: true }),
   async (req, res) => {
+    const { suspended, emailTemplate } = req.body as SuspendUserPayload;
     const { id } = req.params;
     const user = await db.user.get(id);
     if (!user) {
-      res.status(404);
-      return res.json({ errors: ["not found"] });
+      return res.status(404).json({ errors: ["not found"] });
     }
-    if (req.body.suspended === undefined) {
-      res.status(400);
-      return res.json({ errors: ["suspended field required"] });
-    }
-    logger.info(
-      `set user ${id} (${user.email}) suspended ${req.body.suspended}`
-    );
+    const { email } = user;
 
-    await db.user.update(id, { suspended: !!req.body.suspended });
+    logger.info(`set user ${id} (${email}) suspended ${suspended}`);
+    await db.user.update(id, { suspended });
+
+    suspendUserStreams(req, id, suspended).catch((err) => {
+      logger.error(
+        `error suspending user streams id=${id} email=${email} err=${err}`
+      );
+    });
+
+    if (suspended && emailTemplate === "copyright") {
+      const {
+        frontendDomain,
+        supportAddr,
+        sendgridTemplateId,
```

Comment:
Same as above, we have the `sendgridTemplateId` defined anyway, so only copyright with this Id is possible

Reply:
Hmm, we have a template for the outer portion of the email, but we still need to provide the content for it including the subject, text, etc, right? So that's what I meant with the copyright template. We can rethink any of this later when we either add more sendgrid templates or more "text templates", which is what I made for the copyright one.

I don't think automatically sending an email about copyright infringiment makes sense tho, so that's why I wanted to specify what email is supposed to be sent. There are various reasons why we might want to suspend an user, and forgetting that it automatically sends a copyright email and sending it when it was not the case seems worse to me than forgetting about including the param here and not sending the email sometimes (which won't happen with the automatic tooling anyway).

Reply:
OK I changed the code a bit to make my intent clearer, and also made sure that we always email users, using just a simple message when the copyright is not specified.

LMK what you think!
https://github.com/livepeer/livepeer-com/pull/989/commits/1d1d529ea4c38e2cd4a8d11b52138eecc9012256

Reply:
Oh ok I see! Much clearer and makes sense!

Reply:
Default email template powered by GitHub Autopilot®! 😂